pkg/cmd/roachprod/docker/entrypoint.sh

```diff
@@ -9,4 +9,6 @@ aws configure set aws_secret_access_key $(cat /secrets/aws_secret_access_key)
 # library to work
 aws configure set region us-east-1
 az login --service-principal -u $(cat /secrets/azure_user_id) -p $(cat /secrets/azure_password) -t $(cat /secrets/azure_tenant_id)
-exec /usr/local/bin/roachprod $@
+/usr/local/bin/roachprod "$@"
```

Comment:
Without `exec`, this would still send SIGTERM to `roachprod` when a POD is shutdown, right?

Reply:
The pod will be shut down in any case, after the scripts exits. I removed `exec` to let the next line run.